tests/test_base.py

```diff
@@ -61,3 +61,42 @@ def test_tmdb_get_params_bool(self):
         search.movie(query=MOVIEQUERY3, include_adult='true')
         total_results2 = search.total_results
         self.assertEqual(total_results1, total_results2)
+
+    # _get_params (1) adds the api key as a parameter and (2) turns all booleans into their string form.
+    # this test ensures these pieces of functionality when api_key is defined globally.
+    def test_get_params_only_api_key(self):
+        tmdb.API_KEY = API_KEY
+        tmdb_object = tmdb.base.TMDB()
+        expected = {'api_key': API_KEY}
+        actual = tmdb_object._get_params(params={})
+        self.assertEqual(actual, expected)
+
+    # confirm _get_params will error when API_KEY is not defined
+    def test_get_params_api_key_undefined(self):
+        tmdb_no_api_defined = tmdb.base.TMDB()
+        tmdb.API_KEY = None
+        with self.assertRaises(tmdb.base.APIKeyError):
+            tmdb_no_api_defined._get_params(params={})
+        tmdb.API_KEY = API_KEY
```

Comment:
everytime I change the API_KEY to `None`, I "reset" it like so to preserve the other tests